dothething/dothething/ClipsView.swift

```diff
@@ -35,71 +34,101 @@ struct ClipsView: View {
     @Binding var toggle: Bool
     
     var body: some View {
-        VStack {
-            HStack {
-                // back button
-                Button(action: {
-                    print("Back button tapped")
-                    clipsViewModel.backButtonPressed()
-                    toggle.toggle()
-                }) {
-                    Text("Back")
-                }                                                                                                                                                                                                                                                                                                                                                                                                                                   
-                .padding(.leading)
-                .frame(maxWidth: .infinity, alignment: .leading)
+        GeometryReader { geometry in
+            VStack {
+                ZStack {
+                    HStack {
+                        Button(action: {
+                            print("Back button tapped")
+                            clipsViewModel.backButtonPressed()
+                            toggle.toggle()
+                        }) {
+                            Image(systemName: "chevron.left")
+                                .font(.system(size: 24))
+                                .foregroundColor(.accentColor)
+                        }
+                        .padding(.leading)
+                        Spacer()
+                    }
+                    
+                    Text("domino")
+                        .font(.custom("Montserrat-Medium", size: 27))
+                        .foregroundColor(Color.accentColor)
+                        .tracking(8)
+                        .multilineTextAlignment(.center)
+                }
                 
-                // upload button
-                Button(action: {
-                    clipsViewModel.uploadButtonPressed()
-                }) {
-                    Text("Upload")
+                Text("rally")
+                    .font(.custom("Montserrat-Medium", size: 17))
+                    .padding(.bottom)
+                    .tracking(4)
+
+                if !clipsViewModel.errorText.isEmpty && clipsViewModel.clips.isEmpty {
+                    Text(clipsViewModel.errorText)
+                        .padding(.top)
+                        .foregroundColor(Color(UIColor.systemGray))
+                } else if clipsViewModel.isLoading {
+                    ProgressView()
+                        .progressViewStyle(.circular)
+                        .padding(.top)
                 }
-                .opacity(clipsViewModel.uploadDisabled ? 0.5 : 1)
```

Comment:
This section of the code displays an upload button in the top-right corner of the ClipsView. The changes in the new version remove the upload button and replace it with a "place a domino" button at the bottom of the view.